packages/vm/src/backend.rs

```diff
@@ -125,8 +125,8 @@ pub trait Storage {
 /// We can use feature flags to opt-in to non-essential methods
 /// for backwards compatibility in systems that don't have them all.
 pub trait BackendApi: Copy + Clone + Send {
-    fn canonical_address(&self, human: &HumanAddr) -> BackendResult<CanonicalAddr>;
-    fn human_address(&self, canonical: &CanonicalAddr) -> BackendResult<HumanAddr>;
+    fn canonical_address(&self, human: &str) -> BackendResult<Vec<u8>>;
```

Comment:
This is just for backwards compatibility, but
```suggestion
    fn canonical_address(&self, addr: &str) -> BackendResult<Vec<u8>>;
```

Reply:
Why this change? Does it make a difference? This is just the (unused) variable name in a trait. We use `human` for human readable address input and `canonical` for canonical address input.

Reply:
Yeah, this is irrelevant. Forget about it.